package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@web-eid/web-eid-library",
+  "name": "@web-eid/web-eid.js",
```

Comment:
This name is published in GitLab Package Registry and people may already use it as-is, so this should be reverted.

We will change the package name when we publish it to npmjs.com, https://www.npmjs.com/package/web-eid (currently Martin's old package).